From c0af3be2fcee73c1a61f58572ecef1ef3f385015 Mon Sep 17 00:00:00 2001
From: Sergey Poznyakoff <gray@gnu.org>
Date: Thu, 15 Apr 2021 08:37:29 +0300
Subject: Bugfix in mimeview testsuite

* mimeview/tests/bf.c (write_byte): Fix type of the character buffer.
---
 mimeview/tests/bf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mimeview/tests/bf.c b/mimeview/tests/bf.c
index 6ccce848c..2e6b37e4a 100644
--- a/mimeview/tests/bf.c
+++ b/mimeview/tests/bf.c
@@ -68,7 +68,7 @@ write_string (FILE *fp, char *arg)
 int
 write_byte (FILE *fp, char *arg)
 {
-  int c;
+  char c;
   if (strlen (arg) == 3
       && (arg[0] == '\'' || arg[0] == '"'))
     c = arg[1];
-- 
cgit v1.2.1

